core/src/main/scala/org/dbpedia/extraction/mappings/DraftMappingExtractor.scala

```diff
@@ -0,0 +1,102 @@
+package org.dbpedia.extraction.mappings
+
+import org.dbpedia.extraction.destinations.{Dataset, DBpediaDatasets, Quad}
+import org.dbpedia.extraction.ontology.Ontology
+import org.dbpedia.extraction.util.{ExtractorUtils, Language}
+import org.dbpedia.extraction.wikiparser._
+
+import scala.language.reflectiveCalls
+
+/**
+  * Extracts template variables from template pages (see http://en.wikipedia.org/wiki/Help:Template#Handling_parameters)
+  */
+class DraftMappingExtractor(context: {
```

Comment:
everything else looks good, only rename this file / class to `InfoboxMappingsExtractor` to merge